app/Http/Controllers/ControllerDash.php

```diff
@@ -178,7 +178,9 @@ public function showProfile($year = null, $month = null) {
                 Config::get('scheddy.base').'/api/userSessions/'.$user_id,
                 ['headers' => [
                     'Authorization' => 'Bearer '.Config::get('scheddy.api_key')
-                ]]
+                ],
+                'http_errors' => false
+                ]
```

Comment:
By adding this, the user will have no way to know that there is an issue with scheddy. Below in the if statement, if the status is not 200 then we should return a status message letting the user know something went wrong with scheddy.

Reply:
Agreed - we should have like we did with EA where we had a " :warning: fetch failed" or whatever.
Also, we could link to https://scheddy.ztlartcc.org/my_sessions - this will become a thing with v1.2.0 on Thursday, which provides the same functionality and can serve as a backup

Reply:
I've taken another look at the code and I'm confident that suppressing http errors is the right thing to do here. This is simply preventing the blade from showing a 500 error when something unexpected happens. We already have infrastructure in-place to handle what you are both suggesting. Take a look at the $appointments_successful variable and how it is used both in this controller and in the blade. When the API call fails, this variable remains false and triggers the display of the message "Could not load training appointments" to the user.

Reply:
Ah, I missed that completely. It is catching that. Looks good!